.github/workflows/release-impl.yml

```diff
@@ -278,7 +281,8 @@ jobs:
         env:
           new_tag: ${{ steps.new_tag.outputs.tag }}
 
-      - uses: actions/upload-artifact@v4
+      - name: Store the imcremental CHANGELOG
```

Comment:
typo: incremental

Reply:
I *keep* making that error, and autocorrect only finds some of then.